hack/cleanup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -eu
+set -u
```

Comment:
I would not remove "fail on error" option. If you'd like a given command to always succeed, please add "|| true" to it.

For example:
crio wipe -f || true

Reply:
done